lib/ui.ex

```diff
@@ -151,7 +180,6 @@ defmodule Ui do
     end
   end
 
-  defp line do
-    "---------\n"
-  end
+  defp line(3), do: "\n-------------\n"
```

Comment:
Done! I tried it before but somehow struggled to find the right formula so that it would look consistent. But I tried again and this time succeeded.